<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DModificarUsuarios</class>
 <widget class="QDialog" name="DModificarUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lModificarUsuario">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>MODIFICAR USUARIO</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="gvFotoModificar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>601</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lNombre">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu nombre: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lApellidos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu apellido: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellidos">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lCorreo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu correo electrónico: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCorreoElectronico">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lContrasena">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu contraseña: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leContrasena">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>470</y>
     <width>96</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLatitud">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu latitud: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLatitud">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leLongitud">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lLongitud">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Introduce tu longitud: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
